### DISCUSSION TITLE
Progress step component -- Disabled states can't be changed

### DISCUSSION THREAD

**Discussion by andoyle** [twilio-labs/paste#3411] [Help: Figma] [answered]
Hey, I need to use the Progress Step component but can't seem to change any of the disabled states to other states. The first two steps I can change to Current, Incomplete, etc. but the three disabled steps I can't change to anything else. 

![Screenshot 2023-08-14 at 15 41 32](https://github.com/twilio-labs/paste/assets/69856100/e44618d4-8ad2-420f-a52a-1ce1982aefc4)
![Screenshot 2023-08-14 at 15 41 37](https://github.com/twilio-labs/paste/assets/69856100/e5ac01bc-36e8-42ac-8dbb-541a67fde6d4)
![Screenshot 2023-08-14 at 15 41 43](https://github.com/twilio-labs/paste/assets/69856100/4d8d0c8c-37f5-4923-b294-d02cbebf5e39)

**Comment by loreina** [comment]
Hi @andoyle,

Thanks for catching this! It looks like the disabled variant is missing in the base component and needs to be remapped. I'll patch this up this week and follow up here. Appreciate your patience!

**Comment by loreina** [comment, marked as answer]
Hi @andoyle,

A fix for Progress Steps was just published in the Paste Components library. Thanks for your patience!
